test/test_spark.py

```diff
@@ -526,7 +526,11 @@ def do_test_spark_run_does_not_default_env_to_os_env(self, use_mpi, use_gloo):
 
     """
     Test that horovod.spark.run fails with os.environ as env with mpi.
+    
+    TODO: figure out why this test sometimes causes hangs in CI
```

Comment:
yet another one of these ...

Reply:
Yeah, I suspect there is something going on with processes not shutting down between tests that's leading to these errors.  It appears to be unique to our environment, as I cannot repro locally.  I will try and make it a priority to figure out what's going on after the 0.20 release.